cbrain_task/nii2mnc/portal/nii2mnc.rb

```diff
@@ -73,7 +73,7 @@ def after_form #:nodoc:
     params = self.params
     cb_error "Missing voxel type"     if params[:voxel_type].blank?
     cb_error "Missing space ordering" if params[:space_ordering].blank?    && !self.tool_config.is_at_least_version("2.0.0")
-    cb_error "Missing voxel int sign" if params[:voxel_int_signity].blank? && params[:voxel_type] =~ /^(byte|short|int)$/
+    cb_error "Missing voxel int sign" if params[:voxel_int_signity].blank? && params[:voxel_type] =~ /\A(default|byte|short|int)\z/
```

Comment:
My bad. You're totally right. I had not noticed that the symbols were different. It happens so often that we test something with

```ruby
blahblah if xyz.blank? || xyz.has_bad_property
```

that I didn't notice we were not actually checking the same xyz.